cmd/config-v10.go

```diff
@@ -68,17 +69,18 @@ func initConfig() (bool, error) {
 		srvCfg.Notify.NATS["1"] = natsNotify{}
 		srvCfg.Notify.PostgreSQL = make(map[string]postgreSQLNotify)
 		srvCfg.Notify.PostgreSQL["1"] = postgreSQLNotify{}
-		srvCfg.rwMutex = &sync.RWMutex{}
 
 		// Create config path.
 		err := createConfigPath()
 		if err != nil {
 			return false, err
 		}
-
+		// hold the mutex lock before a new config is assigned.
+		serverConfigMu.Lock()
 		// Save the new config globally.
 		serverConfig = srvCfg
 
```

Comment:
Does it mean that the assignment with the mutex locks should be in 3 consecutive lines